docs/diy/assembling.md

```diff
@@ -6,12 +6,12 @@ This guide mainly focuses on the parts listed in the [Hardware Buy guide](../qui
 + ESP-32
 + 433 MHz transmitter
 + Soldering station
-+ Hookup wire (Optional depending if the pins line up)
++ Hookup wire (optional, depending on whether the pins line up)
```

Comment:
```suggestion
+ Hookup wire (Optional, depending on whether the pins line up)
```
Taking out this capitalization seems awkward the rest looks good

Reply:
Well optional is an adjective. Adjectives and nouns are usually lowercase unless they mark the beginning of a stance or are a proper noun, none of which are the case here.

Aka I see no reason why not to change it.

Reply:
IMO being linguisticly correct is not the goal here, it's understanding. To me `(Optional) is a serpate thought and `, depending on whether the pins line up)`  explains that secondary thought. Again I'm just nitpicking at this point.

Reply:
I guess that makes sense